app/src/main/java/org/jellyfin/mobile/fragment/WebViewFragment.kt

```diff
@@ -157,6 +157,8 @@ class WebViewFragment : Fragment() {
             override fun onReceivedError(view: WebView, request: WebResourceRequest, error: WebResourceErrorCompat) {
                 val description = if (WebViewFeature.isFeatureSupported(WebViewFeature.WEB_RESOURCE_ERROR_GET_DESCRIPTION)) error.description else null
                 Timber.e("Received WebView error at %s: %s", request.url.toString(), description)
+                if ("${request.url}" == appPreferences.instanceUrl)
```

Comment:
Why not?
```suggestion
                if (request.url == appPreferences.instanceUrl)
```

Reply:
or `request.url.toString()`

Reply:
Operator '`==`' cannot be applied to '`Uri!`' and '`String?`'
`request.url` is an `Uri`, I can change to `request.url.toString()` or keep `"${request.url}"`.

Reply:
> or `request.url.toString()`

Ok